packages/toolkit/scripts/build.ts

```diff
@@ -97,18 +97,21 @@ const entryPoints: EntryPointOptions[] = [
     folder: '',
     entryPoint: 'src/index.ts',
     extractionConfig: 'api-extractor.json',
+    globalName: 'RTK',
   },
   {
     prefix: 'rtk-query',
     folder: 'query',
     entryPoint: 'src/query/index.ts',
     extractionConfig: 'api-extractor.query.json',
+    globalName: 'RTK.QUERY',
   },
   {
     prefix: 'rtk-query-react',
     folder: 'query/react',
     entryPoint: 'src/query/react/index.ts',
     extractionConfig: 'api-extractor.query-react.json',
+    globalName: 'RTK.QUERY.REACT',
```

Comment:
:hand: Let's use the same name for both RTKQ entries.  Users should only be loading one or the other anyway, and I think it's better to keep the same name either way.

Reply:
Yes, I'll fix this in the next commit and when we have settled where the entry will be placed in the global namespace.

Reply:
this is fixed in the latest commit